metadata/client.go

```diff
@@ -1536,6 +1536,22 @@ type User struct {
 	fetchPropertiesFn
 }
 
+func (e Entity) Variant() string {
```

Comment:
shouldn't this functions be ordered along with their superclass? So shouldn't the e entity variant be with th Entity class?

Reply:
yeah good call out. I'll place them closer to their parent.

Reply:
done.